src/actions/onboardingActions.js

```diff
@@ -160,17 +161,21 @@ const finishRegistration = async ({
   dispatch,
   getState,
   userInfo,
+  mnemonic,
   privateKey,
   address,
   isImported,
+  enableBiometrics,
 }: {
   api: SDKWrapper,
   dispatch: Dispatch,
   getState: GetState,
-  userInfo: Object, // TODO: add back-end authenticated user model (not people related ApiUser)
+  userInfo: Object, // TODO: add back-end authenticated user model (not people related ApiUser),
```

Comment:
:thinking:

Reply:
@poocart

Reply:
@grilix Could you specify what exactly is wrong here? I assume the comma at the end of the comment, right?

Reply:
if it's about TODO then it's my legacy :D we can actually try to build that type in SDK and retrieve it from there so each time back-end changes something they can also include in SDK and we have it in front-end

cc @msazdova @damsal7

Reply:
the SDK has all the required types in TS format. We could see how we could use them in Flow